### PR TITLE
fix: breadcrumb separator display

### PR DESCRIPTION
**Pull request by zkwolf**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/breadcrumb/BreadcrumbSeparator.tsx [hunk 1]
```diff
@@ -16,10 +17,15 @@ export default defineComponent({
...
     const { prefixCls: customizePrefixCls } = this;
+    const { separator, class: className, ...restAttrs } = this.$attrs;
     const getPrefixCls = this.configProvider.getPrefixCls;
     const prefixCls = getPrefixCls('breadcrumb', customizePrefixCls);
 
     const children = getSlot(this);
-    return <span class={`${prefixCls}-separator`}>{children || '/'}</span>;
+    return (
+      <span class={[`${prefixCls}-separator`, className]} {...restAttrs}>
+        {(children.length > 0 && children) || '/'}
```
Comment (John60676) [inline components/breadcrumb/BreadcrumbSeparator.tsx:27]: `children.length > 0 && children`  这段是不是写反了，而且 `&& children` 应该可以不写，`getSlot ` 貌似能确保返回的是数组

Comment (zkwolf) [reply components/breadcrumb/BreadcrumbSeparator.tsx:27]: getSlot就是确保返回数组啊，这里正确的逻辑是如果数组是空的返回'/'，否则返回children

Comment (John60676) [reply components/breadcrumb/BreadcrumbSeparator.tsx:27]: 哦，看错了 😂

Comment (tangjinzhou) [reply components/breadcrumb/BreadcrumbSeparator.tsx:27]:
```js
children.length > 0 ? children : '/'
```
改成这种吧

**Review by tangjinzhou — CHANGES_REQUESTED** [review]

**Pull request merged**